### PR TITLE
Code enhancement

### PR DESCRIPTION
**Pull request by yceruto**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | no |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

Fixing some minor PHPDoc, removed unused imports and variables.

### CODE REVIEW

**Comment by HeahDude** [comment]
Any clean up should be sent against 2.7

File: src/Symfony/Bridge/Twig/Tests/Extension/TranslationExtensionTest.php [hunk 1]
```diff
@@ -87,17 +87,17 @@ public function getTransTests()
 
             // transchoice
             array('{% transchoice count from "messages" %}{0} There is no apples|{1} There is one apple|]1,Inf] There is %count% apples{% endtranschoice %}',
-                'There is no apples', array('count' => 0)),
+                'There is no apples', array('count' => 0), ),
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/Tests/Extension/TranslationExtensionTest.php]: these comma should be reverted, we don't do this in the code base

**Comment by yceruto** [comment]
Oops! sorry, closing and reworking in 2.7

**Comment by HeahDude** [comment]
@yceruto No need for closing anymore thanks to github new feature allowing to change the target branch, just rebase ;)

**Comment by yceruto** [comment]
@HeahDude Yes, I know, but I've many conflicts and I prefer start the work again, thanks.

File: src/Symfony/Bridge/PhpUnit/ErrorAssert.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ public static function assertErrorsAreTriggered($expectedType, $expectedMessages
         $triggeredMessages = array();
 
         try {
-            $prevHandler = set_error_handler(function ($type, $message, $file, $line, $context) use ($expectedType, &$triggeredMessages, &$prevHandler) {
```
Comment (stof) [inline src/Symfony/Bridge/PhpUnit/ErrorAssert.php]: this change is broken. The variable is passed by reference to the closure, so changing the reference is not an unused variable.

File: src/Symfony/Bridge/Doctrine/Tests/Fixtures/SingleAssociationToIntIdEntity.php [hunk 3]
```diff
@@ -14,7 +14,6 @@
 use Doctrine\ORM\Mapping\Id;
 use Doctrine\ORM\Mapping\Column;
 use Doctrine\ORM\Mapping\Entity;
-use Doctrine\ORM\Mapping\OneToOne;
```
Comment (stof) [inline src/Symfony/Bridge/Doctrine/Tests/Fixtures/SingleAssociationToIntIdEntity.php]: this annotation **is** used

File: src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/TemplatePathsCache/templates.php [hunk 4]
```diff
@@ -1,3 +1,5 @@
-<?php return array (
+<?php
+
+return array(
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/TemplatePathsCache/templates.php]: These files are fixture files used as comparison for the generated ones, so this change is wrong (it makes tests fail)

**Pull request closed without merging**